api/src/main/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimRepository.java

```diff
@@ -91,4 +91,13 @@ default Claim findClaim(UUID claimId) {
      * @return claim for given reference.
      */
     Optional<Claim> findByReference(String reference);
+
+    /**
+     * gets all claims.
+     *
+     * @return list of all existing claims.
+     */
+    @Override
+    List<Claim> findAll();
```

Comment:
we don't need findAll() here, repository should already have this method. Reuse the same.